pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -177,7 +177,7 @@ func TestPostFilter(t *testing.T) {
 				"node1": framework.NewStatus(framework.Unschedulable),
 			},
 			wantResult: framework.NewPostFilterResultWithNominatedNode(""),
-			wantStatus: framework.NewStatus(framework.Unschedulable, "preemption: 0/1 nodes are available: 1 No preemption victims found for incoming pod."),
+			wantStatus: framework.NewStatus(framework.Unschedulable).WithError(errors.New("0/1 nodes are available: 1 No preemption victims found for incoming pod.")),
```

Comment:
Use `framework.AsStatus`

Reply:
Nope, it's intended, explained at https://github.com/kubernetes/kubernetes/pull/110900#discussion_r914228213